caffe2/serialize/versions.h

```diff
@@ -58,13 +58,21 @@ constexpr uint64_t kMaxSupportedFileFormatVersion = 0x6L;
 // 6. Write version string to `./data/version` instead of `version`.
 
 #if ENABLE_UPGRADERS
-// This is set to 7 from 3 due to a different interpretation of what
-// file format version is. Whenever there is new upgrader introduced,
+// [12/15/2021]
+// kProducedFileFormatVersion is set to 7 from 3 due to a different
+// interpretation of what file format version is.
+// Whenever there is new upgrader introduced,
 // this number should be bumped.
+// The reasons that version is bumped in the past:
 //     1. aten::div is changed at version 4
 //     2. aten::full is changed at version 5
 //     3. torch.package uses version 6
-constexpr uint64_t kProducedFileFormatVersion = 0x7L;
+//
+// [01/24/2022]
+// We bump the version number to 8 to update aten::linspace
```

Comment:
Shouldn't this comment be bullet point 4 in the above list?

Reply:
Wouldn't this be version number 7, not 8? Below it's `linspace_0_7`, too

Reply:
I updated the comment to make it little more clear. We use version 7 before this change as a way to mark we are switching to a new design. And `linspace_0_7` actually implies that there was a version bump to 8 for operator linspace and this upgrader will cover the versions from 0 to 7 inclusively.

Reply:
Right, that makes sense